sentry_sdk/_compat.py

```diff
@@ -1,5 +1,6 @@
 import sys
 import contextlib
+from datetime import datetime, timezone
```

Comment:
If we try to import `timezone` here it'll raise an ImportError in Python 2 -- I'd only import `datetime` here...
```suggestion
from datetime import datetime
```
...and then `timezone` in the `else` branch of the `if PY2:` condition that starts on line 85, e.g. before the `urllib.parse` import.